core/src/main/scala/org/genivi/sota/core/rvi/SotaServices.scala

```diff
@@ -29,18 +30,21 @@ final case class ClientServices( start: String, abort: String, chunk: String, fi
 /**
  * RVI message from client to initiate a package download.
  */
-final case class StartDownload(vin: Vehicle.Vin, packages: List[Package.Id], services: ClientServices )
+final case class StartDownload(vin: Vehicle.Vin, updateId: UUID, services: ClientServices)
 
 /**
  * RVI parameters of generic type for a specified service name.
  */
 final case class RviParameters[T](parameters: List[T], service_name: String )
 
+final case class OperationResult(id: String, resultCode: Int, resultText: String)
```

Comment:
For Genivi, the operation ids are in the uploaded 'update image'. We don't need to parse it for now, just store it in db.